src/conversion/model-to-view-converters.js

```diff
@@ -252,9 +254,11 @@ export function removeAttribute( attributeCreator ) {
  *
  * @param {module:engine/view/element~Element|Function} elementCreator View element, or function returning a view element, which will
  * be used for wrapping.
+ * @param {Function} [nameToConsumable] Optional function used to map conversion event name to consumable name. By default it
+ * uses {engine/conversion/model-to-view-converters~eventNameToConsumableType} function.
  * @returns {Function} Set/change attribute converter.
  */
-export function wrapItem( elementCreator ) {
+export function wrapItem( elementCreator, nameToConsumable = eventNameToConsumableType ) {
```

Comment:
I don't know what the code does and why but from used techniques it seems to be completely fine. You really looked too late at this, @pjasiun ;) It's just a callback, like many others all around our code. I don't see any particularly wrong with this one.